redbot/cogs/trivia/data/lists/worldflags.yaml

```
@@ -140,6 +140,7 @@ What country is represented by this flag? https://i.imgur.com/LgwBULL.png:
 What country is represented by this flag? https://i.imgur.com/Ls07jJq.png:
 - Democratic Republic of the Congo
 - Dem. Rep. of Congo
+- Drc
```

```suggestion
- DRC
- D.R.C.
```
In order to maintain aliasing consistency within the various trivia lists, you should probably add an additional alias that separates this abbreviation with full stop characters ("DRC" + "D.R.C."). You should also apply this change to the other abbreviating aliases you have added in this PR.